libcaf_net/caf/net/transport_worker_dispatcher.hpp

```diff
@@ -65,48 +68,48 @@ class transport_worker_dispatcher {
 
   template <class Parent>
   error handle_data(Parent& parent, span<byte> data, id_type id) {
-    auto it = workers_by_id_.find(id);
-    if (it == workers_by_id_.end()) {
-      // TODO: where to get node_id from here?
+    auto worker = find_by_id(id);
+    if (!worker) {
+      // TODO: where to get id_type from here?
```

Comment:
What is the TODO about? We do have an ID inside this function.

Reply:
This was a typo. I actually meant `node_id`.
When there is no worker for incoming traffic, the dispatcher instantiates a new one. For that it needs the `node_id` and the `id_type` of the incoming traffic.